q2_demux/_summarize/assets/quality-plot.html

```diff
@@ -63,8 +63,23 @@ <h5 class="text-center">Forward Reads</h5>
   <div id="reverseContainer" class="col-xs-12 col-md-6">
     <h5 class="text-center">Reverse Reads</h5>
   </div>
+  <div class="row">
+    <div class="col-lg-12">
+      <a href="forward-index-quality-scores.csv">Download forward quality as CSV</a>
```

Comment:
How about we change the label of this link to "Download forward parametric seven-number summaries as CSV"? These aren't the subsampled quality scores in this CSV, just the seven-number summaries for all sequence base positions.